surya/detection/__init__.py

```diff
@@ -12,11 +12,10 @@
 
 from surya.detection.loader import DetectionModelLoader
 from surya.detection.parallel import FakeExecutor
-from surya.detection.processor import SegformerImageProcessor
 from surya.detection.util import get_total_splits, split_image
-from surya.detection.schema import TextDetectionResult
+from surya.detection.schema import TextDetectionResult, TextBox
 from surya.settings import settings
-from surya.detection.heatmap import parallel_get_lines
+from surya.detection.heatmap import parallel_get_boxes, parallel_get_lines
 
 
 class DetectionPredictor(BasePredictor):
```

Comment:
We should have an inlinepredictor and a detectionpredictor.  The inlinepredictor can inherit from detectionpredictor, and just override the loader.  We can deal with the merging of lines in marker - not in surya